### PR TITLE
Updates opencv-python and remediates rest of Golang issues

### PR DESCRIPTION
**Pull request by alexk307**
**Describe the change**

Upgrades opencv python bindings.

```
├──────────────────────────────────┼─────────────────────┼──────────┤          ├───────────────────┼───────────────┼─────────$
│  opencv-contrib-python (METADATA) │ GHSA-cxjf-x6jp-p7mc │ HIGH     │          │ 4.6.0.66          │ 4.8.1.78      │ opencv-contrib-python bundled libwebp binaries in wheels   │
│                                  │                     │          │          │                   │               │ that are vulnerable to CVE-2023-4863                       │
│                                  │                     │          │          │                   │               │ https://github.com/advisories/GHSA-cxjf-x6jp-p7mc          │
├──────────────────────────────────┼─────────────────────┤          │          │                   │               ├───────────────────────────────$
│ opencv-python (METADATA)         │ GHSA-qr4w-53vh-m672 │          │          │                   │               │ opencv-python bundled libwebp binaries in wheels that are  │
│                                  │                     │          │          │                   │               │ vulnerable to CVE-2023-4863                                │
│                                  │                     │          │          │                   │               │ https://github.com/advisories/GHSA-qr4w-53vh-m672          │
├──────────────────────────────────┼─────────────────────┼──────────┤          ├───────────────────┼───────────────┼─────────$
```

Upgrades to Go 1.22.9 and bumps up vulnerable deps.

### CODE REVIEW

File: build/python/backend/requirements.txt [hunk 1]
```diff
@@ -27,8 +27,8 @@ nested-lookup==0.2.22
 numpy==1.22.1
 olefile==0.46
 oletools==0.60.1
-opencv-python==4.6.0.66
-opencv-contrib-python==4.6.0.66
+opencv-python==4.8.1.78
```
Comment (alexk307) [inline build/python/backend/requirements.txt:30]: This wasn't in their report, but it showed up for me when I scanned with `trivy`

**Review by cameron-dunn-sublime — APPROVED** [review]

File: build/python/backend/Dockerfile [hunk 2]
```diff
@@ -134,6 +134,7 @@ RUN cd /strelka/ && \
     git \
     libtool \
     make \
+    unrar \
```
Comment (alexk307) [inline build/python/backend/Dockerfile:137]: FYSA: needed to remove `unrar` after using it as it has a vuln (added after your reviewed @cameron-dunn-sublime).

Comment (cameron-dunn-sublime) [reply build/python/backend/Dockerfile:137]: Does that interrupt https://github.com/sublime-security/strelka/blob/7229988c119e5e3c9b496f64d9c73c174515f589/src/python/strelka/scanners/scan_rar.py#L17?

Comment (alexk307) [reply build/python/backend/Dockerfile:137]: good catch! let's see 😟

Comment (alexk307) [reply build/python/backend/Dockerfile]: It's not in their report, and we're on the latest available in our package manager. If they come back and complain about it I'll have to compile from source...

**Pull request merged**